lua/cfc_ulx_commands/timed_punishments/server/data.lua

```diff
@@ -0,0 +1,119 @@
+local function escapeArg( arg )
+    if arg == SQL_NULL then
+        return "NULL"
+    elseif type( arg ) == "number" then
+        return arg
+    else
+        return sql.SQLStr( arg )
+    end
+end
+
+local function queryFormat( query, ... )
```

Comment:
Maybe a better name can be used here? My only issue is that it reformats the query AND runs it... Maybe you could break up the query into its own function or something? I think just renaming it would be better though.